### PR TITLE
Draft: Add e2e tests with playwright

### PR DESCRIPTION
**Pull request by not4Pedro**
🚨Please review the [guidelines for contributing](https://github.com/taranis-ai/.github/blob/master/CONTRIBUTING.md) to this repository.

## Summary by CodeRabbit

- **New Features**
  - Integrated Git LFS for better handling of large SQL and dump files.
  - Added a script to easily recreate the local Playwright testing environment.
  - Updated the test database with versioning and integrity checks.

- **Documentation**
  - Added instructions for Playwright test execution and video generation.

- **Tests**
  - Implemented new Playwright test cases to automate web application interactions.
  - Introduced a suite of local Playwright tests for enhanced end-to-end testing scenarios.

### CODE REVIEW

**Comment by not4Pedro** [comment]
resolves #78

File: resources/e2e_test_db.sql
Comment (b3n4kh) [inline resources/e2e_test_db.sql:1]: maybe add this file as via LFS

File: src/core/tests/test_local_playwright.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
+from playwright.sync_api import Playwright, sync_playwright, expect
+import time
+
+
+def highlight_element_with_css(page, locator, duration):
```
Comment (b3n4kh) [inline src/core/tests/test_local_playwright.py:5]: That looks like an awesome approach!

File: src/core/tests/auth.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        {
+          "name": "main",
+          "value": "{\"user\":{\"id\":\"\",\"name\":\"\",\"organization_name\":\"\",\"permissions\":[]},\"vertical_view\":false,\"drawerVisible\":true,\"itemCountTotal\":0,\"itemCountFiltered\":0,\"coreAPIURL\":\"http://127.0.0.1:5000/api\",\"gitInfo\":\"\",\"buildDate\":\"2023-11-18T16:32:09.587Z\",\"sentryDSN\":\"\",\"notification\":{\"message\":\"\",\"type\":\"\",\"show\":false}}"
+        },
+        {
+          "name": "authenticator",
+          "value": "{\"jwt\":\"\",\"login_uri\":\"/login\",\"logout_uri\":\"/logout\",\"external_login_uri\":false,\"external_logout_uri\":false,\"user\":{},\"sub\":\"\",\"exp\":0}"
+        },
+        {
+          "name": "ACCESS_TOKEN",
```
Comment (b3n4kh) [inline src/core/tests/auth.json:16]: We should try to replace this and try to inject the token from the pytest fixture.

File: resources/e2e_test_db.sql
Comment (not4Pedro) [reply resources/e2e_test_db.sql:1]: This is a really good idea. Thanks

File: src/core/tests/auth.json (hunk 2 above)
Comment (not4Pedro) [reply src/core/tests/auth.json:16]: We could definitely consider this. However, this is ONLY needed when generating the code for tests using the Codegen tool.

**Comment by not4Pedro** [comment]
@coderabbitai review

**Review by b3n4kh — APPROVED** [review]

**Pull request merged**